src/CloudNative.CloudEvents.Avro/AvroEventFormatter.cs

```diff
@@ -41,30 +41,35 @@ public class AvroEventFormatter : CloudEventFormatter
         private const string DataName = "data";
 
         private static readonly string CloudEventAvroMediaType = MimeUtilities.MediaType + MediaTypeSuffix;
-        private static readonly RecordSchema avroSchema;
-        private static readonly DefaultReader avroReader;
-        private static readonly DefaultWriter avroWriter;
+        private readonly RecordSchema avroSchema;
```

Comment:
I don't understand why this has been changed to be parsed multiple times rather than as part of type initialization. Is that actually required?

Reply:
You're correct, this modification isn't needed. I've applied your suggestion of making this field a public static property.